lpc55_sign/src/bin/lpc55_sign.rs

```diff
@@ -545,3 +517,69 @@ fn check_plain_image(_image: &[u8]) -> Result<()> {
     check!(OK, "Nothing to check for plain image");
     Ok(())
 }
+
+fn read_certs(paths: &[PathBuf]) -> Result<Vec<Vec<u8>>> {
+    Ok(paths
+        .iter()
+        .map(std::fs::read)
+        .collect::<Result<Vec<Vec<u8>>, _>>()?)
+}
+
+fn write_to_file(path: &PathBuf, bytes: &[u8]) -> Result<()> {
```

Comment:
This seems identical to [`std::fs::write`](https://doc.rust-lang.org/std/fs/fn.write.html)

Reply:
RIght you are, thanks. Fixed in 884d197.